tests/Common/ChartDrawingUiTest/ChartDrawingUiTest.csproj

```diff
@@ -56,9 +56,6 @@
     <Prefer32Bit>true</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Xaml.Behaviors, Version=1.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Microsoft.Xaml.Behaviors.Wpf should not need to be deleted.
Make sure the build goes through.